addons/website_sale/views/templates.xml

```diff
@@ -197,7 +197,6 @@
                             </del>
                         </t>
                         <span class="h6 mb-0" t-if="template_price_vals['price_reduce'] or not website.prevent_zero_price_sale" t-esc="template_price_vals['price_reduce']" t-options="{'widget': 'monetary', 'display_currency': website.currency_id}"/>
-                        <span class="h6 mb-0" t-elif="any(ptav.price_extra for ptav in product.attribute_line_ids.product_template_value_ids)">&amp;nbsp;</span>
```

Comment:
It seems this line recently added as a bug fix to prevent displaying that message when product attribute has an extra price
PR: https://github.com/odoo/odoo/pull/184052
So I don't think this is a bug :fearful: 

@lvsz can you check this one?

Reply:
Initially, I also believed this might not be a bug. However, upon further investigation, I noticed that even when opening the product, the text "Not Available For Sale" is displayed. Additionally, there is no "Add to Cart" button available, making it impossible to add the product to the cart. What do you think? :thinking: @kcv-odoo @lvsz